src/typeDefinitions/filterUsersByRole.d.ts

```diff
@@ -0,0 +1,12 @@
+export type UserArray = {
+  user: {
+    id: string;
+  };
+  roles: string[];
+};
```

Comment:
can we use the predefined user type here?

Reply:
would have used it but it had more type declaration and I am not even using those variables in my function. hence didn't use them